Software/Latex Docs/Software Steps/Software Steps.tex

```diff
@@ -203,75 +199,39 @@ \subsection{Setting up serial communication}
 
 \noindent Below is a link to help with the above steps if you need additional help:
 \begin{itemize}
-	\item \href{https://spellfoundry.com/2016/05/29/configuring-gpio-serial-port-raspbian-jessie-including-pi-3/}{https://spellfoundry.com/2016/05/29/configuring-gpio-serial-port-raspbian-jessie-including-pi-3/}. 
+	\item \href{https://spellfoundry.com/2016/05/29/configuring-gpio-serial-port-raspbian-jessie-including-pi-3/}{https://spellfoundry.com/2016/05/29/configuring-gpio-serial-port-raspbian-jessie-including-pi-3/}.
 \end{itemize}
 \noindent Once you've completed the above steps, reboot the Pi.
 \begin{itemize}
 	\item[] \colorbox{lightgray}{sudo reboot}
 \end{itemize}
 
-\subsection{Downloading the Rover Code}
+\subsection{Building the Rover Code to work with your ROS installation}
 On the Raspberry Pi, open up a terminal \textbf{(ctl + alt + t)} and then type the following commands\footnote{In this document, terminal commands will be \colorbox{lightgray}{highlighted}.}:
 
-\begin{itemize}
-	\item[] Move to your home directory: \colorbox{lightgray}{cd /home/pi}
-	\item[] Clone the rover code repository: \colorbox{lightgray}{git clone https://github.com/nasa-jpl/osr-rover-code osr}
-	\item[] Move into the directory you just cloned: \colorbox{lightgray}{cd osr}
-	\item[] Pull to make sure you have the latest code: \colorbox{lightgray}{git pull}
-\end{itemize}
-
-\subsection{Building the Rover Code to work with your ROS installation}
-Now, you must build the rover code packages so that ROS can run them on your Raspberry Pi.  To do this, run the following commands in a terminal:
+Create a catkin workspace which will contain all ROS compilation and source code files:
```

Comment:
Lines 212 and 215 are the same instruction, just worded differently.  I like the new wording (line 212), but it should be in the 'itemize' list, replacing the old instruction (line 215).